benchexec/tools/cetfuzz.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for CETFUZZ
+    An Automatic Test Suit Generator
+    URL: https://zenodo.org/records/10065857
```

Comment:
Is this really the preferred URL for your tool? Don't you have a webpage where people could for example see new versions etc.?

Reply:
https://gitlab.com/Sarathkrishnan/cetfuzz
It our tool updated the repository.